### ISSUE TITLE
float16 support with BatchNorm layer

### ISSUE THREAD

**Issue by Golbstein** [keras-team/keras#13516]
Hey

Will we ever be able to train keras model intuitively with float16?
2.3.1 version still doesn't support float16 training for BatchNormalization layer.

**Comment by vladHoloArch** [comment]
I'm afraid no, because you lox
